templates/admin/website-toolbar.html.twig

```diff
@@ -0,0 +1,9 @@
+<div style="position: fixed; bottom: 0; left: 0; background-color: black;padding: 5px;">
```

Comment:
setting the background color based on the status would definitely make this easier to spot in the screenshot. this way it kinda looks like the loading status of the browser 😇

Reply:
Done ;)